locations/spiders/shell.py

```diff
@@ -12,14 +22,28 @@ def parse_item(self, item, location):
         amenities = location["amenities"]
         fuels = location["fuels"]
         apply_category(Categories.FUEL_STATION, item)
-        if "shop" in amenities:
+        if "twenty_four_hour" in amenities:
+            item["opening_hours"] = OpeningHours()
+            item["opening_hours"].add_days_range(DAYS, "00:00", "23:59")
+        if "shop" in amenities or "selectshop" in amenities:
```

Comment:
See https://github.com/alltheplaces/alltheplaces/pull/5767#issue-1875398065

I think Shell Select is big enough to be its own POI. I don't think "shop" is, so I'd prefer it mapped to `shop=yes`.